src/App.js

```diff
@@ -1,42 +1,31 @@
 import "./App.css";
-import logo from "./logo.png";
+import "./css/main.css";
+
+import {  Switch,
+          Route,
+          Link, 
+          Routes} from 'react-router-dom';
+import { Nav } from "./Components/Header/Nav";
+import { Footer } from "./Components/Footer/Footer";
```

Comment:
suggestion: just create an index file in the component folder and export all components from it. so it will be like 
                     **import {Nav, Footer} from './Components**

Reply:
okay will implement it.